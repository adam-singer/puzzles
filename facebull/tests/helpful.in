M0 C0 C1 678
M1 C0 C3 996
M2 C0 C2 931
M3 C1 C0 748
M4 C2 C3 469
M5 C2 C0 249
M6 C2 C1 74
M7 C3 C0 667
