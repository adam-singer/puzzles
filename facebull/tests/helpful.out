2889
2 3 4 6 7
